src/console/Microsoft.Health.Fhir.Ingest.Console.Normalization/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/dotnet/runtime:6.0 AS base
+WORKDIR /app
+
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+WORKDIR /src
+COPY ["nuget.config", "."]
+COPY ["src/console/Microsoft.Health.Fhir.Ingest.Console.Normalization/Microsoft.Health.Fhir.Ingest.Console.Normalization.csproj", "src/console/Microsoft.Health.Fhir.Ingest.Console.Normalization/"]
+COPY ["src/console/Microsoft.Health.Fhir.Ingest.Console.Common/Microsoft.Health.Fhir.Ingest.Console.Common.csproj", "src/console/Microsoft.Health.Fhir.Ingest.Console.Common/"]
+COPY ["src/lib", "src/lib"]
+COPY . .
```

Comment:
Does this recursively copy everything into the image? If so, do you still need lines 9-11?

Reply:
I removed this line and added only the necessary files outside of src/console and src/lib.